Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableEvent.cs

```diff
@@ -91,12 +91,29 @@ public ReceiverData AddReceiver(Type type)
             return data;
         }
 
+        /// <summary>
+        /// True when event types have been compiled once by GetEventTypes
+        /// </summary>
+        private static bool eventListsCompiled = false;
```

Comment:
Why static?  This would mean one interactable could have it's events compiled and others might not yet be

Reply:
The function searches loaded assemblies for a list of objects of type ReceiverBase. The list isn't tailored to each Interactable - it's global. So once it's compiled once, we can just return the cached list regardless of who's asking.